Documentation/observability/metrics.rst

```diff
@@ -426,9 +426,19 @@ Name                                     Labels
 ======================================== ================================================== ========== ========================================================
 ``controllers_runs_total``               ``status``                                         Enabled    Number of times that a controller process was run
 ``controllers_runs_duration_seconds``    ``status``                                         Enabled    Duration in seconds of the controller process
+``controllers_group_runs_total``         ``status`` ``group_name``                          Enabled    Number of times that a controller process was run, labeled by controller group name (configurable)
 ``controllers_failing``                                                                     Enabled    Number of failing controllers
 ======================================== ================================================== ========== ========================================================
 
+The ``controllers_group_runs_total`` metric reports the success and failure
+count of each controller within the system, labeled by controller group name
+and completion status. Due to the large number of controllers, enabling this
+metric is on a per-controller basis. This is configured using an allow-list
+which is passed as the ``controller-group-metrics`` configuration option,
```

Comment:
In all of these descriptions that you added, can you change `option` to `flag` here, to make it clear to the user that this is something they need to pass to the binary? The word `option` might make users think this is a helm value that they can look for, especially in the description for clustermesh and kvstoremesh where the `prometheus.controllerGroupMetrics` helm value isn't listed. [3]

```suggestion
which is passed as the ``controller-group-metrics`` configuration flag,
```